src/main/java/DukeException.java

```diff
@@ -0,0 +1,23 @@
+public class DukeException {
+
+    public String message;
+    public boolean isValid;
+
+    public DukeException(String message){
```

Comment:
Is it better to place a white space between the close bracket and the open curly brace?